docs/ini-file-deduction.md

```diff
@@ -0,0 +1,32 @@
+# Deduction of the Core INI Format
+
+Since the INI-file format isn't specified yet, the features of INI-files depend heavily on the used INI-parser and its configuration.
+
+This is the attempt to create an overview of commonly used parsers and their interpretation of different INI-dialects to find a common feature-set which may be used as base for the strict specification of the EditorConfig-INI parser.
+
+| Parser | Inline comments | Multiline Values | Escape Characters | Encoding Support | Empty Values | Trimmed Names | Example-Reference |
```

Comment:
Can we rename "Trimmed Names" to "Trimmed Keys" and swap its location with "Empty Values" so that keys are on the left and values are on the right?

Reply:
It looks like "empty values" is a common feature, yes? Does that mean we should also support it? I'm not understanding the use case. I mean, if the existence of the key is sufficient, it sounds like it would be better expressed as a Boolean, right?

```ini
foo=true
```

vs.

```ini
foo=
```

And if you really just want a placeholder for a property, add a comment!

```ini
# foo=todo
```